df_chat/drf/filters.py

```diff
@@ -0,0 +1,29 @@
+from django_filters import rest_framework as filters
+
+from ..models import Message
+
+class MessageFilter(filters.FilterSet):
+    """
+    Filter-set class for `Message` list API.
+    All the filters that will be accessible by the filter backend
+        are defined here.
+    """
+    # For `datetime` range filter on the model's `created` field.
+    created_gte = filters.DateTimeFilter(
+        field_name="created", lookup_expr="gte")
+    created_lte = filters.DateTimeFilter(
+        field_name="created", lookup_expr="lte")
+    
+    # For filter on the model's related user's `username` field.
+    # TODO: Maybe a better/cleaner implementation for this?
+    username = filters.CharFilter(
+        field_name="room_user__user__username", lookup_expr="contains")
+    
+    # For text search on the model's `body` field.
+    # ! This is not efficient. Text search is better handled otherwise. 
```

Comment:
I don't think we will have some advanced text search in this package for now. 

If we will, I think, it's better to create an issue rather than adding a comment.